### PR TITLE
Connectors on Operator to connect to Confluent Cloud

### PR DESCRIPTION
**Pull request by rohit2b**

### CODE REVIEW

File: ccloud-integration/README.rst [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+   ::
+
+     kubectl create secret tls ca-pair-sslcerts \
+       --cert=$TUTORIAL_HOME/ca.pem \
+       --key=$TUTORIAL_HOME/ca-key.pem
+
+Provide authentication credentials
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Confluent Cloud provides an user and API key for both Kafka and Schema Registry. You'll need to get these,
```
Comment (arodoni) [inline ccloud-integration/README.rst:73]:
```suggestion
Confluent Cloud provides you and API key for both Kafka and Schema Registry.
```

File: ccloud-integration/README.rst [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+and configure Confluent Operator to use them to configure Connect and KsqlDB to connect.
```
Comment (arodoni) [inline ccloud-integration/README.rst:74]:
```suggestion
Configure Confluent Operator to use the API key when setting up Connect and KsqlDB to connect.
```

File: ccloud-integration/README.rst [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+#. Create a Kubernetes secret object for Confluent Cloud Kafka access.
```
Comment (arodoni) [inline ccloud-integration/README.rst:76]: There is only one step. Can we remove the ordered list?

File: ccloud-integration/README.rst [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+   ::
+   
+     kubectl create secret generic control-center-user \
+      --from-file=basic.txt=$TUTORIAL_HOME/creds-control-center-users.txt
+
+=========================
+Deploy Confluent Platform
+=========================
+
+Edit the `confluent-platform.yaml` deployment YAML, and add your respective Confluent Cloud URLs in the following places:
```
Comment (arodoni) [inline ccloud-integration/README.rst:101]:
```suggestion
Edit the ``confluent-platform.yaml`` deployment YAML, and add your respective Confluent Cloud URLs in the following places:
```

File: ccloud-integration/README.rst [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+- `<cloudSR_url>`
```
Comment (arodoni) [inline ccloud-integration/README.rst:103]:
```suggestion
- ``<cloudSR_url>``
```

File: ccloud-integration/README.rst [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+- `<cloudKafka_url>`
```
Comment (arodoni) [inline ccloud-integration/README.rst:104]:
```suggestion
- ``<cloudKafka_url>``
```

**Comment by rohit2b** [comment]
Incorporated the review comments!

**Review by arodoni — APPROVED** [review]

**Pull request merged**